src/vhdlConvertor/packageHeaderParser.cpp

```diff
@@ -35,15 +35,38 @@ std::unique_ptr<HdlNamespace> VhdlPackageHeaderParser::visitPackage_declaration(
 	//       END ( PACKAGE )? ( identifier )? SEMI
 	// ;
 	ph = create_object<HdlNamespace>(ctx);
+	ph->name = VhdlLiteralParser::getIdentifierStr(ctx->identifier(0));
```

Comment:
Moved this up here so it's with the other property for `ph`.  This also matches more closely how the VhdlPackageParser code looks.